electron-builder.yml

```diff
@@ -6,7 +6,6 @@ nodeGypRebuild: false
 publish: null
 files:
   - lib
-  - "!lib/gui/app"
```

Comment:
Sorry if this is a stupid question, but if it was only the settings that were affected, should this be modified to _just_ pull in the settings-related files, rather than the whole of `lib/gui/app`? :man_shrugging: